code/modules/holodeck/HolodeckControl.dm

```diff
@@ -281,6 +281,11 @@
 	for(var/obj/holo_obj in holographic_objs)
 		holo_obj.alpha *= 0.8 //give holodeck objs a slight transparency
 
+	if(HP.loop_ambience)
```

Comment:
Why not just set `loop_ambience` to this unconditionally? All the if is doing is preventing it from being null, which if you _really_ need you can do with an or.